client-sdk/go/crypto/signature/ed25519/signer.go

```diff
@@ -26,6 +26,10 @@ func (w wrappedSigner) Reset() {
 	w.signer.Reset()
 }
 
+func (w wrappedSigner) Unwrap() coreSignature.Signer {
+	return w.signer
```

Comment:
could alternatively make the signer field public

could alternatively store the consensus signer and wrap on demand